ui/admin/app/components/form/role/manage-custom-scopes/index.js

```diff
@@ -5,27 +5,57 @@
 
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
 
 export default class FormRoleManageCustomScopesIndexComponent extends Component {
+  // =attributes
+
+  @tracked showRemoveOrgModal = false;
+  @tracked selectedOrg;
```

Comment:
Is the boolean needed if you are also keeping track of the `selectedOrg`?

Reply:
good point, the bool has been removed.